packages/button/docs/Button.stories.tsx

```diff
@@ -200,9 +214,19 @@ const StickerSheetTemplate: Story<{ isReversed: boolean }> = ({
         </StickerSheet>
       )}
 
-      <StickerSheet isReversed={isReversed} heading="Working / With Icon">
+      <StickerSheet
+        isReversed={isReversed}
+        heading="Working / With Icon / With Badge"
+      >
         <StickerSheet.Header
-          headings={["Working", "Working (Focus)", "Icon Left", "Icon Right"]}
+          headings={[
+            "Working",
+            "Working (Focus)",
+            "Icon Left",
+            "Icon Right",
+            "Badge Left",
+            "Badge Right",
```

Comment:
```suggestion
            "Icon Left with Badge",
            "Icon Right with Badge",
```

Something like that. `Badge Left` made me think the badge should show on the left of the text.